k9mail/src/main/res/xml/global_preferences.xml

```diff
@@ -415,12 +415,12 @@
 
         <org.openintents.openpgp.util.OpenPgpAppPreference
             android:persistent="false"
-            android:key="crypto_app"
+            android:key="openpgp_provider"
```

Comment:
We don't use a SharedPreferences to back this but do the saving manually in `saveSettings`, so the key is independent. We *could* go for consistency and name it the same everywhere, but that also masks how these values are really from different namespaces, leading to wrong assumptions about the code structure.